ui/apps/platform/src/Containers/MainPage/Body.tsx

```diff
@@ -281,6 +291,22 @@ function Body({ hasReadAccess, isFeatureFlagEnabled }: BodyProps): ReactElement
                             <Redirect to={`${policiesBasePath}/${match.params.policyId}`} />
                         )}
                     />
+                    {isFeatureFlagEnabled('ROX_PLATFORM_CVE_SPLIT') && (
+                        <Route
+                            // We _do not_ include the `exact` prop here, as all prior workload-cves routes
+                            // must redirect to the new path. Instead we match against all subpaths.
+                            path={`${vulnerabilitiesWorkloadCvesPath}/:subpath*`}
+                            // Since all subpaths and query parameters must be retained, we need to do
+                            // a search and replace of the subpath we are redirecting
+                            render={({ location }) => {
+                                const newPath = location.pathname.replace(
+                                    vulnerabilitiesWorkloadCvesPath,
+                                    vulnerabilitiesAllImagesPath
+                                );
+                                return <Redirect to={`${newPath}${location.search}`} />;
+                            }}
```

Comment:
This is worthwhile to share at team meeting, because it looks like what I did not know how to do for a previous redirect. Thankfully, then have been rare.

@bradr5 Important, but not urgent to evaluate if React Router 6 has same or different idiom.

Reply:
Luckily I think this will be relatively easy to update in v6. It will be more or less changing `render` -> `element` and swapping out `<Redirect />` with `<Navigate />`.